HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -2105,20 +2105,70 @@ def self.apply_hvac(runner, hpxml_bldg, weather, convert_shared_systems, unit_nu
       heat_pump.airflow_defect_ratio_isdefaulted = true
     end
 
-    # Fan power
+    # Fan model type
+    hpxml_bldg.heating_systems.each do |heating_system|
+      next unless heating_system.heating_system_type == HPXML::HVACTypeFurnace
+      next unless heating_system.fan_model_type.nil?
+      next if (not heating_system.distribution_system.nil?) && (heating_system.distribution_system.air_type == HPXML::AirTypeGravity)
+
+      if (not heating_system.attached_cooling_system.nil?) && (not heating_system.attached_cooling_system.compressor_type.nil?)
+        # Based on RESNET DX Modeling Appendix
+        heating_system.fan_model_type = (heating_system.attached_cooling_system.compressor_type == HPXML::HVACCompressorTypeSingleStage) ? HPXML::HVACFanModelTypePSC : HPXML::HVACFanModelTypeBPM
+      else
+        # HEScore assumption
+        heating_system.fan_model_type = (heating_system.heating_efficiency_afue > 0.9) ? HPXML::HVACFanModelTypeBPM : HPXML::HVACFanModelTypePSC
+      end
+      heating_system.fan_model_type_isdefaulted = true
+    end
+    hpxml_bldg.cooling_systems.each do |cooling_system|
+      next unless cooling_system.fan_model_type.nil?
+
+      if (not cooling_system.attached_heating_system.nil?) && (not cooling_system.attached_heating_system.fan_model_type.nil?)
+        cooling_system.fan_model_type = cooling_system.attached_heating_system.fan_model_type
+        cooling_system.fan_model_type_isdefaulted = true
+      elsif [HPXML::HVACTypeCentralAirConditioner].include? cooling_system.cooling_system_type
+        # Based on RESNET DX Modeling Appendix
+        cooling_system.fan_model_type = (cooling_system.compressor_type == HPXML::HVACCompressorTypeSingleStage) ? HPXML::HVACFanModelTypePSC : HPXML::HVACFanModelTypeBPM
+        cooling_system.fan_model_type_isdefaulted = true
+      elsif [HPXML::HVACTypeMiniSplitAirConditioner].include? cooling_system.cooling_system_type
+        cooling_system.fan_model_type = HPXML::HVACFanModelTypeBPM
+        cooling_system.fan_model_type_isdefaulted = true
+      elsif [HPXML::HVACTypeEvaporativeCooler].include? cooling_system.cooling_system_type
+        # Depends on airflow rate, so defaulted in hvac_sizing.rb
+      end
+    end
+    hpxml_bldg.heat_pumps.each do |heat_pump|
+      next unless heat_pump.fan_model_type.nil?
+
+      if [HPXML::HVACTypeHeatPumpAirToAir].include? heat_pump.heat_pump_type
+        # Based on RESNET DX Modeling Appendix
+        heat_pump.fan_model_type = (heat_pump.compressor_type == HPXML::HVACCompressorTypeSingleStage) ? HPXML::HVACFanModelTypePSC : HPXML::HVACFanModelTypeBPM
+        heat_pump.fan_model_type_isdefaulted = true
+      elsif [HPXML::HVACTypeHeatPumpGroundToAir].include? heat_pump.heat_pump_type
+        if heat_pump.heating_efficiency_cop > 8.75 / 3.2 # HEScore assumption
```

Comment:
This is kept the same as previous assumptions, need to discuss when we have two speed/var speed GSHP capabilities. Not sure for that case should it follow the RESNET assumption dependent on speed number or should it be something else (out of RESNET scope).